ebd-all-minilm/build-apple-darwin.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Usage: ./build-apple-darwin.sh <arch> <arch_name>
+#
+# Parameters:
+#   arch: The architecture to use.
+#   arch_name: The architecture name to use.
+
+set -e
+
+ARCH=${1:-arm64}
+ARCH_NAME=${2:-aarch64}
```

Comment:
would ignore intel for now :)

```suggestion
set -e
```

Reply:
Done :)